test/cadet/achievements/achievement_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule Cadet.Achievments.AchievementTest do
+  alias Cadet.Achievements.Achievement
+
+  use Cadet.ChangesetCase, entity: Achievement
+
+  describe "Changesets" do
+    test "valid changesets" do
+      assert_changeset(
+        %{
+          inferencer_id: 0,
+          id: 0,
+          title: "Hello World",
+          ability: :Core,
+          exp: 0,
+          open_at: Timex.now() |> Timex.to_unix() |> Integer.to_string(),
+          close_at: Timex.now() |> Timex.shift(days: 7) |> Timex.to_unix() |> Integer.to_string(),
```

Comment:
A `DateTime.t()` (as returned by Timex) should work here, I think? No need to stringify them...

Reply:
👍